src/routes/backup/RestoreBackup.js

```diff
@@ -116,6 +116,12 @@ const modal = ({
             { backingImages.map(backingImage => <Option key={backingImage.name} value={backingImage.name}>{backingImage.name}</Option>) }
           </Select>)}
         </FormItem>
+        <FormItem label="Encrypted" {...formItemLayout}>
+          {getFieldDecorator('encrypted', {
+            valuePropName: 'encrypted',
+            initialValue: false,
```

Comment:
Just a check. It seems users need to specify the secret info again when restoring a backup, right? (maybe there are some place to improve this part in the future)

Reply:
Yes, if we need to get the corresponding volume.encrypted this needs to be considered a bit more. If the volume has been deleted we won't get the encrypted property. I can consider setting the initial value for it if needed.